content/graphics-switch-ubuntu.md

```diff
@@ -102,6 +102,8 @@ Now enable the System76 Power extension with the toggle to the right.
 Click the system menu in the top right corner of your screen to access graphics
 switching.
 
+>**Note**: Ubuntu versions higher than 22.04 will not have this menu because the GNOME Shell extension has not been updated for newer GNOME versions.
```

Comment:
I really dislike how this is worded. Better would be:
**Note**: Gnome Shell versions newer than 42 are not supported by the System76 Power extension

Reply:
I feel like the issue is that most folks don't know the version of GNOME that ships with their version of Ubuntu. Maybe this?:

GNOME Shell versions newer than 42 (Ubuntu 22.04 and above) are not supported by the System76 Power extension.

Reply:
Ubuntu 22.04 ships with a Gnome 42 implementation, so:

**Note:** Gnome Shell versions newer than 42 (Ubuntu 22.10 or later) are not supported by the System76 Power extension.

Reply:
New commit changes this to match that.